src/server/worldserver/worldserver.conf.dist

```diff
@@ -812,6 +813,24 @@ HeroicCharactersPerRealm = 1
 CharacterCreating.MinLevelForHeroicCharacter = 55
 
 #
+#    DemonHuntersPerRealm
+#        Description: Limit number of demon hunter characters per account on this realm.
+#        Range:       1-10
+#        Default:     1
+
+DemonHuntersPerRealm = 1
+
+#
+#    CharacterCreating.MinLevelForDemonHunter
+#        Description: Limit creating heroic characters only for account with another
```

Comment:
@Ovahlord 
Nice work - I have some Pandaria scripts Draenor - if you want to look at them and try to implement I thank you.
PS: There is complete files some are just the base!

[Pandaria and Draenor](https://github.com/Ovahlord/TrinityCore/files/378445/Pandaria.and.Draenor.zip)